data/binding/gen.go

```diff
@@ -133,9 +133,10 @@ func (b *prefBound{{ .Name }}) checkForChange() {
 const toStringTemplate = `
 type stringFrom{{ .Name }} struct {
 	base
-
+{{ if .Format }}
 	format string
-	from   {{ .Name }}
+{{ end }}
+	from {{ .Name }}
```

Comment:
I'm sorry what is the nit here? Goimports passed it fine.

Reply:
The nit is that the variable types are no longer aligned because there is an extra newline between them;

<img width="323" alt="Screen Shot 2021-02-09 at 08 59 30" src="https://user-images.githubusercontent.com/1117080/107399049-22c6a280-6ab5-11eb-9ad0-b558b1c79cbf.png">

Reply:
In some instances the "format" string is output, in others it is not.
If we keep it without that space then I will have to emit two different versions of "from" depending on whether format is included. The apprach of having a gap for the consistency seemed cleaner.